src/coreclr/nativeaot/Runtime/gcrhenv.cpp

```diff
@@ -1366,17 +1366,8 @@ bool GCToEEInterface::GetBooleanConfigValue(const char* privateKey, const char*
         return true;
     }
```

Comment:
You can also delete TCHAR definition from src\coreclr\gc\env\gcenv.structs.h. I think it was there to make this work.

Reply:
Seems like some other parts of nativeaot are actually using the definition from there now. I might try to detangle in the future change, but I'm going to leave it for now.